content/ai/concepts/generative-ai/concepts/text/text.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Text'
+Description: Generative AI can create blogs, write ad copy, create new content based on text input, and is great at summarizing and changing the style and tone of text content.
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+  - 'Transformers'
+Tags:
```

Comment:
Only AI is listed on the [Tags list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).
All the other terms should be added to that file in this PR.